scripts/schema-manager/manifest-loader.ts

```diff
@@ -0,0 +1,104 @@
+import { TigrisSchema } from '@tigrisdata/core/dist/types';
+import path from 'path';
+import fs from 'fs';
+import * as Log from './logger';
+
+const SCHEMAS_ROOT = 'models';
```

Comment:
Other options - `/lib/models`, `/tigris/models`, `/lib/data`, `/db`. Looking for opinions here.

Reply:
`/models` looks good to me

Reply:
Or we could have `/models/tigris/` That way we will not need the `-db` and `-schema` suffix, what do you think?

Reply:
this looks good, then we can create everything.